utils/build-llvm-local.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
+
 ##===- utils/build-llvm-local.sh - Build LLVM on local machine --*- Script -*-===##
-#
-# This file licensed under the Apache License v2.0 with LLVM Exceptions.
-# See https://llvm.org/LICENSE.txt for license information.
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-#
+# 
+# Copyright (C) 2022, Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
```

Comment:
no reason to change the license here

Reply:
Thanks, addressed in [f18a8d6](https://github.com/Xilinx/mlir-aie/pull/570/commits/f18a8d66b255ed857b56380a62fe27e5ba046e25).